docs/FAQs.md

```diff
@@ -0,0 +1,146 @@
+# FAQ'S
+Contents of this document.
+* [General](#general)
+* [Deployment and config](#deployment-and-config)
+* [API](#api)
+* [Handler](#handler)
+* [Model-archiver](#model-archiver)
+
+## General
+Relevant documents.
+- [Torchserve readme](https://github.com/pytorch/serve#torchserve)
+
+### Does Torchserve API's  follow some REST API standard?
+Torchserve API's are compliant with the [OpenAPI specification 3.0](https://swagger.io/specification/).
+
+### What is not Torchserve?
+Torchserve is not a complete web application to serve end to end business use cases. Hence a lot of security aspects should be made available via 3rd party wrapper components in front of Torchserve.
+
+### What's difference between Torchserve and a python web app using web frameworks like Flask, Django?
+The Flask app and Torchserve are completely different except the fact that both support handling HTTP requests however using different engines [netty or python apis].
```

Comment:
Can you please add comments to issues / specific customer ask for this question. Seems a bit too general

Reply:
added relevant issues